src/content/content_manager.cc

```diff
@@ -325,7 +325,7 @@ void ContentManager::shutdown()
             if (dir != nullptr) {
                 auto dirEnt = fs::directory_entry(dir->getLocation());
                 if (dirEnt.is_directory()) {
-                    auto t = to_time_t(dirEnt.last_write_time());
+                    auto t = std::chrono::duration_cast<std::chrono::seconds>(dirEnt.last_write_time().time_since_epoch());
```

Comment:
to_time_t should be obsolete and removed.

Reply:
Removed.